### ISSUE TITLE
[Question]: perform action an a button entity

### ISSUE THREAD

**Issue by ZEROX7** [LesTR/homeassistant-minimalistic-area-card#62]
### What happened?

i would like for the doorbell button to perform action open.lock an a specific device but i cant seem to get it to work

### Card Version

1.2.12

### HomeAssistant Version

2024.9.2

### Relevant log output

```shell
type: custom:minimalistic-area-card
title: Gang
area: gang
icon: hue:room-hallway
show_icon: true
hide_unavailable: false
shadow: false
state_color: true
tap_action:
  action: navigate
  navigation_path: /dashboard-cast/gang
entities:
  - light.gang
  - light.lightstrip
  - entity: input_button.haustur_offnen
    tap_action:
      action: perform_action
      perform_action: lock.open
      target: lock.haustur
  - lock.wohnungstur
layout_options:
  grid_columns: 2
  grid_rows: 2

```

**Comment by LesTR** [comment]
yeah, it's a bug, but I have no simple way to fix it. This comes from [custom-card-helpers](https://github.com/custom-cards/custom-card-helpers) which is not maintained and it's a side effect of renaming `service` to `action` in HA.

The `perform_action` event is not handled in CCH - https://github.com/custom-cards/custom-card-helpers/blob/master/src/handle-action.ts#L44

As a workaround, you can use the old syntax,  and I'll try to find a way to get rid of the custom-card-helper as it is not maintained, but that will not be simple.

I use this for testing and it works:
```yaml
tap_action:
   action: call-service
   service: light.toggle
   target:
      entity_id: light.toilet_ventilator_light
```       
you can try something like:
```yaml
tap_action:
   action: call-service
   service: lock.open
   target: 
      entity_id: lock.haustur
```

**Comment by LesTR** [comment]
Looks like this was fixed in one custom-card-helpers fork - https://github.com/dermotduffy/custom-card-helpers/commit/306ee10ee85a46d28618160540e9caa2fcf2f14e

**Comment by LesTR** [comment]
@ZEROX7 this should be fixed in [v1.2.13](https://github.com/LesTR/homeassistant-minimalistic-area-card/releases/tag/v1.2.13).
```yaml
tap_action:
      action: perform-action
      perform_action: light.toggle
      target:
        entity_id: light.toilet_ventilator_light
```
This works now. Just be careful, the action value is `perform-action`, not `perform_action` as you use, and you need to use entity_id target (or another target).

**Comment by ZEROX7** [comment]
jep finally works
entities:
  - entity: input_button.haustur_offnen
    tap_action:
      action: perform-action
      perform_action: lock.open
      target:
        entity_id: lock.haustur
